target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -189,6 +189,7 @@ case "$FIRMWARE" in
 	;;
 "ath10k/pre-cal-pci-0000:00:00.0.bin")
 	case $board in
+    dlink,dir-842-c2|\
```

Comment:
Sorry, I was interrupted when I did initial review.
Indent is wrong here (spaces instead of tab).